src/test/java/org/broadinstitute/consent/http/service/dao/DarCollectionServiceDAOTest.java

```diff
@@ -151,22 +149,18 @@ void testCreateElectionsForDarCollectionWithMultipleDatasetsForAdmin() throws Ex
     assertTrue(
         createdVotes.stream().anyMatch(v -> v.getType().equals(VoteType.AGREEMENT.getValue())));
 
-    // Make sure that we DID create elections for the additional DAR/Dataset combination
-    List<Election> additionalCreatedElections =
-        electionDAO.findLastElectionsByReferenceIds(List.of(dar2.getReferenceId()));
-    assertFalse(additionalCreatedElections.isEmpty());
   }
 
   /**
    * This test covers the case where: - User is a chairperson - Collection has 1 DAR/Dataset
    * combinations - Elections created should be for the DAR/Dataset for the user
    */
   @Test
-  void testCreateElectionsForDarCollectionChair() throws Exception {
+  void testCreateElectionsForDarByUserChair() throws Exception {
     DarCollection collection = setUpDarCollectionWithDacDataset();
-    Optional<DataAccessRequest> dar = collection.getDars().values().stream().findFirst();
-    assertTrue(dar.isPresent());
-    Integer datasetId = dar.get().getDatasetIds().get(0);
+    DataAccessRequest dar = collection.getDars().values().stream().findFirst().orElse(null);
+    assertNotNull(dar);
```

Comment:
You can skip the assert and by using `orElseThrow()`
```suggestion
    DataAccessRequest dar = collection.getDars().values().stream().findFirst().orElseThrow();
```